pages/design/atomic-template-empty.md

```diff
@@ -50,7 +50,39 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- (e.g. WCAG Techniques or links with background information mentioned in Applicability, Expectations or Assumptions)
+<!-- Add an introduction, if you want to add further information not available on the pages you are linking to in this section. -->
+
+### Understanding WCAG
+
+<!-- Add links to understanding documents for all success criteria the rule maps to, either directly or through a composite rule. Remove if not applicable -->
+
+- <!-- Example of a WCAG SC. REMOVE THIS LINK.--> [WCAG 2.1 - Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)
+
+### Related WCAG Techniques
+
+<!-- Add links to related techniques. Remove if not applicable -->
+
+- <!-- Example of a WCAG Technique. REMOVE THIS LINK. --> [G108: Using markup features to expose the name and role, allow user-settable properties to be directly set, and provide notification of changes](https://www.w3.org/WAI/WCAG21/Techniques/general/G108)
+
+### Standards and Definitions
```

Comment:
Is this supposed to include all links like "document", "shadow including root", … that are used somewhere in the rule?